src/plugins/data/server/index.ts

```diff
@@ -223,6 +223,7 @@ export {
   TabbedAggColumn,
   TabbedAggRow,
   TabbedTable,
+  calcAutoIntervalLessThan,
```

Comment:
Now that we are exporting this publicly and it's not an implementation detail of TimeBuckets, perhaps we should relocate it with the other date interval utils? `common/search/aggs/utils/date_interval_utils`

Also it might make more sense to expose this under the `data.search.aggs` object (line 257) where the other interval-related helpers are

Reply:
good idea! thank you. Will update it soon